content/blog/2021/06/2021-06-21-student-luminy-project-security.adoc

```diff
@@ -11,16 +11,15 @@ authors:
 - qparra
 - mheyries
 opengraph:
-  image: /images/post-images/2020/08-external-fingerprint-storage-phase-3/opengraph.png
+  image: /images/post-images/2021/2021-06-21-student-luminy-project-security.png
 ---
 
 === Context
 
-Jenkins is the world’s leading open-source automation server used by companies large and small around the globe to implement continuous integration and continuous delivery.
-As such, it is critical that the open source plugins that constitute an integral part of its use as a CI/CD solution don’t expose the systems they are used on to any security risks and vulnerabilities.
+Jenkins is a CI/CD solution and as such, it is critical that the open source plugins that constitute an integral part of it don’t expose the systems they are used on to any security risks and vulnerabilities.
 It is in that context that we worked as an audit/code review team to track and report such flaws and problematic practices.
 
-We worked in collaboration with Jenkins Security team member Wadeck Follonier, as part of an end-of-study project during our last year of an IT and cybersecurity Master’s Degree in the university of Aix-Marseille.
+We worked in collaboration with Jenkins Security team member Wadeck Follonier, as part of an end-of-study project during our last year of the link:https://formations.univ-amu.fr/ME5SIN-PRSIN5AA.html[Master's Degree - Reliability and IT Security] in the university of Aix-Marseille.
```

Comment:
```suggestion
We worked in collaboration with Jenkins Security team member Wadeck Follonier, as part of an end-of-study project during our last year of the link:https://formations.univ-amu.fr/ME5SIN-PRSIN5AA.html[Controller's Degree - Reliability and IT Security] in the university of Aix-Marseille.
```

:trollface:

Reply:
Clearly I'm feeling a little too intense at the moment.  I missed the 

![](https://github.githubassets.com/images/icons/emoji/trollface.png)

in my first read of the comment and started the research to confirm that the university had renamed their "Master's degree" to "Controller's degree".  Thanks Daniel for making me smile!